### PR TITLE
added sonarcloud request

### PR DESCRIPTION
**Pull request by ShellyXueHan**
as per https://github.com/BCDevOps/devops-requests/issues/242

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/sonarcloud_request.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+about: To import GitHub repository on SonarCloud.
+title: ''
+labels: sonarcloud-repo
+assignees: caggles, ShellyXueHan
+
+---
+
+## Step 0
+**Is your repository on `bcgov` GitHub Organization?**
+We have install SonarCloud on `bcgov` GitHub Organization for application teams to use. If your repo is not on bcgov, please migrate first.
```
Comment (jefkel) [inline .github/ISSUE_TEMPLATE/sonarcloud_request.md]:
```suggestion
We have install SonarCloud on `bcgov` GitHub Organization for application teams to use.  Only repositories inside the bcgov GitHub Organization are supported through this request.
```

File: .github/ISSUE_TEMPLATE/sonarcloud_request.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+**Are you the admin user of the GitHub repositories?**
+The setup process require a repo admin level access. You will be assigned as the admin of your SonarCloud repos.
```
Comment (jefkel) [inline .github/ISSUE_TEMPLATE/sonarcloud_request.md]: Could make it a requirement to be the admin of your repo to make the request, and just close/cancel any requests where the requestor is NOT the repo admin?
eg:
```suggestion
The request requires a repository administrator to make the request.  The requestor will be assigned as the SonarCloud repository admin.
> Requests from non-Git repository admins will be closed/cancelled.
```

File: .github/ISSUE_TEMPLATE/sonarcloud_request.md (hunk 1 above)
Comment (ShellyXueHan) [reply .github/ISSUE_TEMPLATE/sonarcloud_request.md]: changes included here: https://github.com/BCDevOps/devops-requests/pull/245

File: .github/ISSUE_TEMPLATE/sonarcloud_request.md (hunk 2 above)
Comment (ShellyXueHan) [reply .github/ISSUE_TEMPLATE/sonarcloud_request.md]: yea that could be more clear!

**Pull request merged**